.github/workflows/installation.yml

```diff
@@ -0,0 +1,110 @@
+name: Installation
+on: [push]
+
+jobs:
+  linux-vm:
+    name: ${{ matrix.python-version }} - ${{ matrix.method }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version:
+          - '2.7'
+          - '3.6'
+          - '3.7'
+        method:
+          - 'python ./install.py'
+          - 'pip install --target /opt/rez .'
+        include:
+          - method: 'python ./install.py'
+            exports: 'PATH=${PATH}:/opt/rez/bin/rez'
+          - method: 'pip install --target /opt/rez .'
+            exports: 'PATH=${PATH}:/opt/rez/bin PYTHONPATH=${PYTHONPATH}:/opt/rez'
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install
+      run: |
+        ${{ matrix.method }}
+        find /opt/rez/ -maxdepth 2
```

Comment:
I'm tempted to suggest to use [check-manifest](https://pypi.org/project/check-manifest/) to test that pip would install the correct files. Though, we would probably have to work a little bit on the `MANIFEST.in` to make it work. But that should be done in another PR IMO if we ever decide to go that way.

Reply:
yeah, sounds like a good idea. Let's get a new PR/issue opened for that